winml/lib/Api/LearningModelSessionOptions.h

```diff
@@ -41,6 +44,8 @@ struct LearningModelSessionOptions : LearningModelSessionOptionsT<LearningModelS
   //
   // The default value here is False so that models are not automatically closed on session creation.
   bool close_model_on_session_creation_ = false;
+
+  std::map<winrt::hstring, uint32_t> named_dim_overrides_;
```

Comment:
Can some comments be added here to explain about named_dim_overrides_? Similar to the other properties.

Reply:
updated!